SweetTweetApp/src/main/java/Services/AuthorizationService.java

```diff
@@ -0,0 +1,29 @@
+package Services;
+
+import posts.Post;
+import posts.PostCollection;
+import users.User;
+import users.UserCollection;
+
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+public class AuthorizationService {
+    public boolean isPostAvailable(String postID, String userID) {
+        try {
+            UserCollection users = new UserCollection();
```

Comment:
Is it right to have `UserCollection` recreated at every method call? 

Haven't read the code thoroughly, but, form a common sense's standpoint, such collection is kind of one-and-only for the whole application and must be served via a singleton (well, a static field is some class, being accessed via a static method)